plonetheme/onegovbear/theme/scss/search.scss

```diff
@@ -3,33 +3,39 @@ $searchbox-liveresults-hover-bg-color: darken($menu-bg-color, 5%) !default;
 $searchbox-liveresults-border-color: darken($menu-bg-color, 20%) !default;
 $searchbox-liveresults-border-radius: 5px !default;
 $search-highlight-bg-color: change-color($secondary-color, $lightness:70%, $saturation:90%);
+$searchbox-width: 295px;
 
 @include declare-variables(
   searchbox-liveresults-bg-color,
   searchbox-liveresults-hover-bg-color,
   searchbox-liveresults-border-color,
-  searchbox-liveresults-border-radius);
+  searchbox-liveresults-border-radius,
+  $searchbox-width);
 
 
 .portal-searchbox-wrapper{
-  float: none;
-  position: relative;
-  top: 0;
-
-  @include screen-medium {
-    top: -52px;
-  }
-
-  @include screen-large {
-    float: right;
-  }
-
+  float: left;
+  width: $searchbox-width;
+  padding: 10px;
+  box-sizing: border-box;
+  margin-left: -$searchbox-width;
+  display: table;
+
+  background: $primary-color; /* Old browsers */
+  background: -moz-linear-gradient(left,  $primary-color 0%, $primary-color 50%, $secondary-color 50%, $secondary-color 100%); /* FF3.6+ */
+  background: -webkit-gradient(linear, left top, right top, color-stop(0%,$primary-color), color-stop(50%,$primary-color), color-stop(50%,$secondary-color), color-stop(100%,$secondary-color)); /* Chrome,Safari4+ */
+  background: -webkit-linear-gradient(left,  $primary-color 0%,$primary-color 50%,$secondary-color 50%,$secondary-color 100%); /* Chrome10+,Safari5.1+ */
```

Comment:
Yes, please add a linear-gradient mixin to ftw.theming.
Is it still necessary to use all vendor prefixes?
The Chrome audit says ``Use normal CSS property names instead of vendor-prefixed ones``.